### PR TITLE
Move scripts from specs repo

### PR DESCRIPTION
**Pull request by sergey-shandar**
https://github.com/Azure/azure-rest-api-specs/pull/5719/checks

### CODE REVIEW

File: src/breaking-change.ts [hunk 1]
```diff
@@ -90,34 +80,42 @@ async function runOad(oldSpec: string, newSpec: string) {
...
- * @param swaggerPath Path to the swagger specification file.
- */
-async function processViaAutoRest(swaggerPath: string) {
-  if (swaggerPath === null || swaggerPath === undefined || typeof swaggerPath.valueOf() !== 'string' || !swaggerPath.trim().length) {
-    throw new Error('swaggerPath is a required parameter of type "string" and it cannot be an empty string.');
-  }
+//main function
+export async function runScript() {
+  // Used to enable running script outside TravisCI for debugging
+  const isRunningInTravisCI = process.env.TRAVIS === 'true';
```
Comment (veronicagg) [inline src/breaking-change.ts]: why is this still needed?

File: src/breaking-change.ts [hunk 2]
```diff
@@ -90,34 +80,42 @@ async function runOad(oldSpec: string, newSpec: string) {
...
+  const outputFolder = path.join(os.tmpdir(), "resolved");
```
Comment (veronicagg) [inline src/breaking-change.ts]: why "resolved"? should it be something like "breakingChangesOutput"?

Comment (sergey-shandar) [reply src/breaking-change.ts]: Honestly, no idea. We may change it after merge.

File: src/breaking-change.ts (hunk 1 above)
Comment (sergey-shandar) [reply src/breaking-change.ts]: probably not

**Pull request merged**